ext/bcmath/tests/bcmod.phpt

```diff
@@ -9,8 +9,12 @@ bcmath.scale=0
 echo bcmod("11", "2"),"\n";
 echo bcmod("-1", "5"),"\n";
 echo bcmod("8728932001983192837219398127471", "1928372132132819737213"),"\n";
+echo bcmod("3.5", "4"),"\n";
+echo bcmod("1071", "357.5"),"\n";
 ?>
 --EXPECT--
 1
 -1
 1459434331351930289678
+3.5
```

Comment:
I'm not sure I get this - how comes bcmod returns non-integer? I always thought module operation always returns integer. Also, 3.5 % 4 returns 3, so I don't see why BC version should be different.